src/node_process_methods.cc

```diff
@@ -177,6 +177,11 @@ static void Rss(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(static_cast<double>(rss));
 }
 
+static void GetConstrainedMemory(const FunctionCallbackInfo<Value>& args) {
```

Comment:
Can you add a fast api for this path? If you need any help, I’m happy to collaborate.

Reply:
Thanks for suggestion, i have updated the code, please help review again.

Reply:
Thanks for suggestion, i just close this PR and submit another one later.